--- glue/perl/Makefile.PL	2009-01-20 10:14:42.000000000 +0100
+++ glue/perl/Makefile.PL.oden	2009-01-20 10:58:19.000000000 +0100
@@ -125,6 +125,14 @@
           autoconf_foo(\$config, qr/s,\@/, qr/\@,/, qr/[^,]+/);
     }
 
+		## Code around an autoconf 2.60 bug
+		## http://lists.gnu.org/archive/html/bug-autoconf/2006-06/msg00127.html
+		## $ grep @PACKAGE_VERSION config.status-2.59 config.status-2.60 
+		## config.status-2.59:s,@PACKAGE_VERSION@,2.09,;t t
+		## config.status-2.60:s,@PACKAGE_VERSION@,|#_!!_#|2.09,g
+		foreach ($apache_includes, $apreq_libname, $version) {
+		    s/\|#_!!_#\|//g;
+		}
 }
 
 
